metrics/web/test/dashboard/domain/entities/metrics/receive_build_day_project_metrics_updates_test.dart

```diff
@@ -0,0 +1,308 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/domain/entities/build_day.dart';
+import 'package:metrics/common/domain/repositories/build_day_repository.dart';
+import 'package:metrics/dashboard/domain/entities/collections/date_time_set.dart';
+import 'package:metrics/dashboard/domain/entities/metrics/build_day_project_metrics.dart';
+import 'package:metrics/dashboard/domain/entities/metrics/build_performance.dart';
+import 'package:metrics/dashboard/domain/usecases/parameters/project_id_param.dart';
+import 'package:metrics/dashboard/domain/usecases/receive_build_day_project_metrics_updates.dart';
+import 'package:mockito/mockito.dart';
+
+import '../../../../test_utils/matchers.dart';
+
+void main() {
+  group("ReceiveBuildDayProjectMetricsUpdates", () {
```

Comment:
It seems like this file is mislocated. Should we place it under the `test/dashboard/domain/usecases`?